docs/api.md

```diff
@@ -216,13 +216,10 @@ curl -v \
 
 ### Response
 
-Successful requests will produce a "200 OK" response with the lockout
-advice in the JSON body:
+Successful requests will produce a "200 OK" response:
 
 ```json
-{
-  "lockout": false
-}
+{}
```

Comment:
Thanks, looks like this was left over in the docs after the "lockout" feature was removed.